### DISCUSSION TITLE
[Feature] List Source under Info -> DETAILS when clicking into photo

### DISCUSSION THREAD

**Discussion by gitterspec** [immich-app/immich#13137] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Right now, it seems impossible to know which external library (or just Uploads) a photo is from, which is bizarre to me unless I'm missing it.  We can see the path when clicking the (i) to the right of the filename, but that still doesn't show the external library name.

Propose adding the Source as an entry under DETAILS, for example "Uploads" or "My Ext Library Name"

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4940

**Comment by gitterspec** [comment]
Sorry I missed that previous feature request.
